src/actions/search-results.js

```diff
@@ -130,20 +134,128 @@ export function updateDisplayedItems (results) {
 }
 
 /**
+<<<<<<< HEAD
+* Action that is called when the 'yes' button on a filter is clicked
+* 1. get all the geo and amenity tags
+* 2. if search results already exist, filter the existing results and update
+* the displayedItems
+* 3. launch a new search to get more items
+*/
+
+export function filterResults () {
+  return (dispatch, getState) => {
+    const { search: { tags, items } } = getState();
+    const geoTags = tags.filter(tag => tag.id.indexOf('geo') > -1);
+    const amenityTags = tags.filter(tag => tag.id.indexOf('amenity') > -1);
+    if (items.length > 0) {
+      const results = items.filter(item => {
+        if (item.packageOffer) {
+          return (
+            // geoTags.some(tag => item.packageOffer.hotel.place.country === tag.displayName) && // e.g. either spain or greece
+            amenityTags.every(tag => item.packageOffer.amenities[tag.id.split(':')[1]]) // and with wifi and kids friendly
+          );
+        } else {
+          return true;
+        }
+      });
+      dispatch(updateDisplayedItems(results));
+    }
+    if (geoTags.length > 0) {
+      dispatch(startSearch());
+    }
+  };
+}
+
+function combinePassengersForQuery (childAgeArray, numberOfChildren, numberOfAdults) {
+  const slicedChildAgeArray = childAgeArray.slice(0, Number(numberOfChildren));
+  const childPassengers = slicedChildAgeArray.map(slicedChildAge => {
+    const date = new Date();
+    const year = date.getFullYear() - Number(slicedChildAge.split(' ')[0]);
+    const month = ('0' + date.getMonth()).slice(-2);
+    const day = ('0' + date.getDate()).slice(-2);
+    return (
```

Comment:
you can just return the object on the same line there's no need to spread it out over three lines!